### PR TITLE
External lib update: libogg, libvorbis, libtheora

### PR DESCRIPTION
**Pull request by J08nY**
Updated:
- libogg to latest version: 1.3.2
- libvorbis to latest version: 1.3.5
- libtheora to latest version 1.1.1

updates #5248

### CODE REVIEW

File: drivers/vorbis/books/Makefile.in [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Makefile.in generated by automake 1.10.2 from Makefile.am.
+# Makefile.in generated by automake 1.15 from Makefile.am.
```
Comment (akien-mga) [inline drivers/vorbis/books/Makefile.in]: I'm not sure we need to keep the Makefile.in actually, maybe we can drop it?

Comment (J08nY) [inline drivers/vorbis/books/Makefile.in]: Oops missed that one, thought I deleted them all.

Comment (akien-mga) [inline drivers/vorbis/books/Makefile.in]: Check `find -name "Makefile.in"`, there are a few other ones left.

**Comment by J08nY** [comment]
Okay, all Makefiles should be gone now.

**Pull request merged**